homeassistant/components/fibaro/binary_sensor.py

```diff
@@ -25,6 +25,7 @@
     "com.fibaro.smokeSensor": ["Smoke", "mdi:smoking", BinarySensorDeviceClass.SMOKE],
     "com.fibaro.FGMS001": ["Motion", "mdi:run", BinarySensorDeviceClass.MOTION],
     "com.fibaro.heatDetector": ["Heat", "mdi:fire", "heat"],
+    "com.fibaro.binarySensor": ["Binary", None, None],
```

Comment:
As the default behaviour is anyway to assign None to icon and device class of the entity this definition is useless. I suggest that you remove this line.

Reply:
The first item of the collection seem to be useless too (no usage of SENSOR_TYPES[stype][0]), do you want that i cleanup this too ?

Reply:
It is enough that you remove the new line which is not needed. Doing refactorings is better done in separate PR's.